README.md

```diff
@@ -11,13 +11,13 @@ Install `gulp-jsdoc` as a development dependency:
 npm install --save-dev gulp-jsdoc3
 ```
 
-Then, use it where config is the only way to pass in jsdoc options. All CLI options are can be specified here,
-the only exception is ink-docstrap is bundled here and used for templating.
+The config is the only way to pass in jsdoc options. All CLI options are can be specified here.
 
 ```javascript
 var jsdoc = require('gulp-jsdoc3');
 
 gulp.task('doc', function (cb) {
+    var config = require('./jsdoc.json');
```

Comment:
Config isn't needed, it should actually be passed in with undefined to use the default implementation. My oversight